File: GMS-client/src/app/page4/KorisnikGetAllResponse.ts

export interface KorisniciGetAllResponse{
  korisnici : KorisniciGetAllResponseKorisnik[];

}

export interface KorisniciGetAllResponseKorisnik {
  id: number
  ime: string
  prezime: string
  teretanaNaziv: string
  gradNaziv: string
  spolNaziv: string
  tezina: number
  visina: number
}

File: GMS-api/GMS/Endpoint/Korisnik/GetAll/KorisnikGetAllResponse.cs
namespace GMS.Endpoint.Drzava.Search
{
    public class KorisnikGetAllResponse
    {
        public List<KorisnikGetAllResponseRow> Korisnici { get; set; }
    }

    public class KorisnikGetAllResponseRow // samo se radi kada je rezultat lista a ne jedan zapis 
    {
        public int ID { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string TeretanaNaziv { get; set; }
        public string GradNaziv { get; set; }
        public string SpolNaziv { get; set; }
        public float Tezina { get; set; }
        public float Visina { get; set; }
    }

}
